docs/user-interface/controls/shadow.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Shadow"
+description: "Learn how to use the .NET MAUI Shadow class, which paints a shadow around a control."
+ms.date: 10/22/2021
+---
+
+# Shadow
+
+<!-- Sample link, if any, goes here -->
+
+The .NET Multi-platform App UI (.NET MAUI) `Shadow` is a control that paints a shadow around a layout or view. The `VisualElement` class has a `Shadow` bindable property, of type `Shadow`, that enables a shadow to be added to any layout or view.
```

Comment:
Is just a details but, Shadow by itself is not a control, is more, a new Bindable Property allowing to paint a shadow around any Layout or View.